runtime-kernel/linux-kernel-rpi64-lts/autobuild/build

```
@@ -0,0 +1,111 @@
+    SRCARCH="x86"
+elif [[ "${CROSS:-$ARCH}" = "arm64" ]]; then
+    SRCARCH="arm64"
+elif [[ "${CROSS:-$ARCH}" = "loongson3" ]]; then
+    SRCARCH="mips"
+elif [[ "${CROSS:-$ARCH}" = "ppc64el" ]]; then
+    SRCARCH="powerpc"    
+fi
+
+# RPi4 kernel build starts.
+
+abinfo "Building kernel for Raspberry Pi 4."
+
+abinfo "Copying config and setting up env..."
+
+cp -v autobuild/config-rpi4 "$SRCDIR"/.config
+
+. autobuild/build-common/set-var
+
+abinfo "Building kernel..."
+make
+
+abinfo "Stripping symbols..."
+. autobuild/build-common/strip-modules
+
+abinfo "Installing modules..."
+mkdir -p "$PKGDIR"/usr
+make INSTALL_MOD_PATH="$PKGDIR"/usr modules_install
+
+abinfo "Copying kernel..."
+# make a local copy of compiled kernel
+mkdir -p "$PKGDIR/usr/lib/aosc-os-arm64-boot/linux-kernel-$version$LOCALNAME/"
+mkdir -p "$PKGDIR/usr/lib/rpi64/kernel/overlays"
+cp -v arch/arm64/boot/Image "$PKGDIR/usr/lib/aosc-os-arm64-boot/linux-kernel-$version$LOCALNAME/"
+cp -v arch/arm64/boot/Image "$PKGDIR/usr/lib/rpi64/kernel/kernel8.img"
+
+export ARCH=$OLDARCH
+rm -rf "$PKGDIR"/usr/lib/firmware
+
+. autobuild/build-common/ext-hdr
+
+export LOCALNAME_RPI4=$LOCALNAME
+unset LOCALNAME
+
+# RPi4 kernel build ends.
+
+# Since `make mrproper` will destroy the installed kernel modules, we need to make a backup then.
+abinfo "Making backup for kernel and kernel modules..."
+tar -cvf rpi4-kernel.tar $PKGDIR
+
+abinfo "Removing all generated files + config + various backup files..."
+make mrproper
+
+# Now, restore the backup.
+abinfo "Restoring the backup..."
+tar -xvf rpi4-kernel.tar --directory=/
+
+# RPi5 kernel build starts.
+
+abinfo "Building kernel for Raspberry Pi 5."
+
+abinfo "Copying config and setting up env..."
+
+cp -v autobuild/config-rpi5 "$SRCDIR"/.config
+
+. autobuild/build-common/set-var
+
+abinfo "Building kernel..."
+make
+
+abinfo "Stripping symbols..."
+. autobuild/build-common/strip-modules
+
+abinfo "Installing modules..."
+mkdir -p "$PKGDIR"/usr
+make INSTALL_MOD_PATH="$PKGDIR"/usr modules_install
+
+abinfo "Copying kernel..."
+# make a local copy of compiled kernel
+mkdir -p "$PKGDIR/usr/lib/aosc-os-arm64-boot/linux-kernel-$version$LOCALNAME/"
+mkdir -p "$PKGDIR/usr/lib/rpi64/kernel/overlays"
+cp -v arch/arm64/boot/Image "$PKGDIR/usr/lib/aosc-os-arm64-boot/linux-kernel-$version$LOCALNAME/"
+cp -v arch/arm64/boot/Image "$PKGDIR/usr/lib/rpi64/kernel/kernel_2712.img"
+
+export ARCH=$OLDARCH
+rm -rf "$PKGDIR"/usr/lib/firmware
+
+. autobuild/build-common/ext-hdr
+
+export LOCALNAME_RPI5=$LOCALNAME
+
+# RPi5 kernel build ends.
+
+# Copies the dtbs and overlays.
+
+# Note: This can be used by both RPi4 and RPi5.
+# Only copy after both RPi4 and RPi5 kernels and modules were copied.
+
+abinfo "Coping new device tree and overlays to /usr/lib/rpi64/kernel..."
+cp -v arch/arm64/boot/dts/broadcom/*.dtb "$PKGDIR/usr/lib/rpi64/kernel/"
+cp -rv arch/arm64/boot/dts/overlays/*.dtb* "$PKGDIR/usr/lib/rpi64/kernel/overlays/"
```

```suggestion
cp -rv "$SRCDIR"/arch/arm64/boot/dts/overlays/*.dtb* "$PKGDIR"/usr/lib/rpi64/kernel/"$version"/overlays/
```

(Note how the quotes are being used.)